### ISSUE TITLE
Fontwahl für die Code-Beispiele

### ISSUE THREAD

**Issue by UweZiegenhagen** [thomas-hilarius-meyer/LaTeX-fuer-Geisteswissenschaftler#66]
Ist euch schon aufgefallen, dass die Schriftart für Code-Beispiele im Fließtext "nicht optimal" ist? Sie ist m.E. viel zu klein?

http://uweziegenhagen.de/lfgw/usepackage.png

In dem Zusammenhang: Kennt ihr den Noto Font von Google? Der könnte für ein einheitliches Schriftbild und andere Alphabete interessant sein.

https://www.google.com/get/noto/

**Comment by thomas-hilarius-meyer** [comment]
Ja, das stimmt...

**Comment by AKielhorn** [comment]
Ist in TeX-Live enthalten und jetzt auch in der Schriftübersicht.
Kann aber auch nur Latin, Greek, Cyrillic.

**Comment by UweZiegenhagen** [comment]
Nein, eben nicht! :-) Schau mal auf die Webseite oder auf https://en.wikipedia.org/wiki/Noto_fonts

Siehe auch http://www.npr.org/sections/codeswitch/2014/08/03/337168933/-no-tofu-doesn-t-equate-to-no-problem-for-google-universal-typeface

**Comment by UweZiegenhagen** [comment]
Hier ein cooles Video: https://vimeo.com/185700918

**Comment by sieversMartin** [comment]
Die Philosophie scheint zu sein, eine große Fontfamilie zu haben. Das LaTeX-Paket, dass es seit Januar gibt, unterstützt aber wohl nur die "großen drei" Skripte.

**Comment by LukasCBossert** [comment]
Schaue mir auch gerade Noto an; die serifenlose überzeugt leider (noch) nicht - keine Tripple-Ligaturen?! - und die Sans ist an sich zu groß. 
Pluspunkt ist natürlich die immense Verfügbarkeit an möglichen Unicodes.

**Comment by UweZiegenhagen** [comment]
Problem ist noch vorhanden, kann das sich jemand anschauen? 
  
![hoehe](https://user-images.githubusercontent.com/1240092/34910456-583b22e8-f8b6-11e7-992a-a4a6491371a3.png)
